Makefile

```diff
@@ -0,0 +1,10 @@
+include bml
+
+DOCKER = $(call required-command,docker)
```

Comment:
Is this used?

Reply:
Ahh, not yet...that's for the TODO item listed below but It should be removed before merging